### PR TITLE
Issue/bio zio filter internal server error

### PR DESCRIPTION
**Pull request by stevenbal**
fixes https://github.com/open-zaak/open-zaak/issues/140

requires https://github.com/maykinmedia/django-loose-fk/pull/11 to be merged and version of django-loose-fk to be bumped

### CODE REVIEW

**Comment by joeribekker** [comment]
@stevenbal Many tests failing here.

**Comment by stevenbal** [comment]
@joeribekker the tests should pass once https://github.com/maykinmedia/django-loose-fk/pull/11 is merged and bumped in openzaak

File: src/openzaak/components/zaken/tests/test_zaakinformatieobjecten.py [hunk 1]
```diff
@@ -175,6 +175,23 @@ def test_filter(self):
...
+            {"informatieobject": f"http://testserver.com{io_url}"},
+            SERVER_NAME="testserver.com",
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(
+            response.data[0]["informatieobject"], f"http://testserver.com{io_url}"
+        )
+
```
[the remark this answers is not in the document]
Comment (stevenbal) [reply src/openzaak/components/zaken/tests/test_zaakinformatieobjecten.py]: I have added a test for it

**Comment by sergei-maertens** [comment]
@stevenbal please make sure to use `openzaak.nl` as hostname if you need to pass URL validation. That one is set in `.travis.yml` environment via `ALLOWED_HOSTS`.

Als rather use the `HTTP_HOST` kwargs instead of `SERVER_NAME`

**Pull request merged**